CRUDRecipeEF.BL.DL/Data/DataSeed.cs

```diff
@@ -1,40 +1,54 @@
 ﻿using System.Collections.Generic;
-using CRUDRecipeEF.BL.DL.Entities;
+using CRUDRecipeEF.BL.DL.DTOs;
+using CRUDRecipeEF.BL.DL.Services;
+using System.Linq;
 
 namespace CRUDRecipeEF.BL.DL.Data
 {
-    public static class DataSeed
+    public class DataSeed : IDataSeed
```

Comment:
I tested the seeding yesterday with yannis and i ran it now again. Everything works fine fetching data from data and using services after it has run etc. I think the error is coming from your side

Reply:
How did you test?